### DISCUSSION TITLE
Problems with Mockend — anyone can help?

### DISCUSSION THREAD

**Discussion by yandex-practicum-ios** [community/community#53289] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi! 

I have a Mockend Pro paid account, linked to all repositories. We needed fake API for education purposes, did everything according to Mockend documentation, but we're keep getting «404 page not found» after clicking https://mockend.com/yandex-practicum-ios/fakenft/

All permissions are also given to Mockend, but mistake is keep happening, support is silent for 3 days.
 
Did anyone had the same problem?

**Comment by mockend-dx** [comment]
Hi,

We located your emails in our spam folder. For some reasons, Gmail has flagged them. Truly sorry for the inconvenience.
I've just responded via email.

**Comment by bornfamousTobia** [comment]
01:33, 24 апреля 2023 г., "mockend-dx" ***@***.***>:
Hi,
We located your emails in our spam folder. For some reasons, Gmail has flagged them. Truly sorry for the inconvenience.
I've just responded via email.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
-- Отправлено из мобильного приложения Яндекс Почты
